TileFormats/BatchTable/README.md

```diff
@@ -42,6 +42,8 @@ A Batch Table is composed of two parts: a JSON header and an optional binary bod
 
 ![batch table layout](figures/batch-table-layout.png)
 
+See [Implementation Notes](#implementation-notes) for details about padding between the json and binary sections.
```

Comment:
json -> JSON.

Should the section just be up here?  Should this be an implementation note?  Is it normative, i.e., part of the actual spec?